class MyException(Exception):
    def __init__(self, val):
        self.val = val